db-testing/src/test/java/com/splicemachine/dbTesting/unitTests/store/T_XA.java

```diff
@@ -266,6 +260,17 @@ void XATest_1(
         REPORT("(XATest_1) finishing");
     }
 
+    private long CreateHeapConglomerate(TransactionController tc, T_AccessRow template_row) throws StandardException {
```

Comment:
Please use first lower case for method names

Reply:
done